core/controllers/suggestion.py

```diff
@@ -109,14 +111,16 @@ def post(self):
             self.user_id,
             self.normalized_payload.get('change'),
             self.normalized_payload.get('description'))
-        
-        if suggestion.suggestion_type == (
-            feconf.SUGGESTION_TYPE_TRANSLATE_CONTENT):
-            suggestion_services.update_translation_contribution_stats_at_submission(suggestion)
-
-        if suggestion.suggestion_type == (
-            feconf.SUGGESTION_TYPE_ADD_QUESTION):
-            suggestion_services.update_question_contribution_stats_at_submission(suggestion)
+
+        if platform_feature_services.is_feature_enabled(
+            platform_feature_list.ParamNames.CONTRIBUTOR_DASHBOARD_ACCOMPLISHMENTS.value):
+            if suggestion.suggestion_type == (
```

Comment:
Not sure I understand why this is gated behind a platform parameter. I thought the platform parameter was only for showing the data but we're updating it behind the scenes (and have always been updating it)? Or do we need to run a job to summarize and generate stats models for the existing contributions?

Reply:
That addresses my last question, but not my first concern. Why do we need a platform parameter here -- are we not planning to update contributions currently?

If we were to run a Beam job it would overwrite the data so I'm not sure why we need to gate it at this point -- we just shouldn't display it until we're ready to do so.